go/tricorder/api.go

```diff
@@ -25,11 +26,30 @@ var (
 // read the same data from that region.
 type Region region
 
-// NewRegion creates a new region with a particular update function
+var (
+	// The default region. Its update function does nothing and returns
+	// the current system time.
+	DefaultRegion *Region
```

Comment:
Sorry, it will be initialized in an init() function. Understand that this PR is only an API proposal so that we can discuss what the API will look like. This particular PR should never be merged.